endpointmanager/pkg/capabilityhandler/validation.go

```diff
@@ -0,0 +1,92 @@
+package capabilityhandler
+
+import "strconv"
+
+var version3plus = []string{"3.0.0", "3.0.1", "4.0.0", "4.0.1"}
+var fhir3PlusJSONMIMEType = "application/fhir+json"
+var fhir2LessJSONMIMEType = "application/json+fhir"
+
+// validationError is the structure for validation errors that are saved in the Validation
+// JSON blob in fhir_endpoints for now.
+type validationError struct {
+	Correct  bool   `json:"correct"`
+	Expected string `json:"expected"`
```

Comment:
It seems valuable to include in the long run for querying, especially if this eventually becomes a separate table, but it doesn't seem like makes queries any easier here since the data being checked is in the same table. But it also wouldn't be much extra effort to add it so if you think it's worthwhile to include now I can definitely update it!

Reply:
i agree that long term it's nice but short term less necessary.